src/pretix/plugins/ticketoutputpdf/views.py

```diff
@@ -252,8 +252,11 @@ def title(self):
     def save_layout(self):
         self.layout.layout = self.request.POST.get("data")
         self.layout.save(update_fields=['layout'])
+        if "name" in self.request.POST:
+            self.layout.name = self.request.POST.get("name")
+            self.layout.save(update_fields=['name'])
```

Comment:
Probably not worth the effort to just call `save` once …
```suggestion
        update_fields = ['layout']
        if "name" in self.request.POST:
            self.layout.name = self.request.POST.get("name")
            update_fields.append('name')
        self.layout.save(update_fields=update_fields)
```

Reply:
yeah that's cleaner